infra/scripts/replace-running-components.sh

```diff
@@ -137,37 +137,29 @@ function enable-debug () {
 
 function replace-component() {
     scp -oUserKnownHostsFile=/dev/null -oStrictHostKeyChecking=no $VIC_DIR/bin/$1 root@$VCH_IP:/tmp/$1 2>/dev/null
-    on-vch mv /sbin/$1 /tmp/old-$1
+    on-vch chmod 755 /tmp/$1
     on-vch mv /tmp/$1 /sbin/$1
-    on-vch chmod 755 /sbin/$1
-    on-vch rm -f /tmp/old-$1
 }
 
 function replace-components () {
-    echo "Killing vic-init (and not taking no for an answer)..."
-    on-vch systemctl kill vic-init
     if [[ $1 == "" ]]; then # replace everything
-        for x in port-layer-server docker-engine-server vicadmin vic-init; do
-            echo "Replacing component $x..."
-            replace-component $x
-        done
+        services="port-layer-server docker-engine-server vicadmin vic-init"
     else
-        for x in $@; do # replace provided list
-            echo "Replacing component $x..."
-            replace-component $x
-        done
+        services="$@"
     fi
-    echo "Starting vic-init.."
-    on-vch systemctl start vic-init
+
+    for x in $services; do
+        echo "Replacing component $x..."
+        replace-component $x
+    done
+
+    # George swears you don't need to do this but if you don't, the new binaries are not launched
```

Comment:
George still swears you don't need to do this and wants logs from it not happening so we can track down WHY.
As written in this PR it's likely because the component processes haven't been killed after the binary is replaced.